content/hardware/04.pro/boards/portenta-x8/tutorials/09.image-flashing/content.md

```diff
@@ -13,26 +13,146 @@ hardware:
 
 ## Instructions
 
+It is recommended to periodically check if your Portenta X8 image version is up to date to ensure you have the latest security updates. Four methods are available for updating your Portenta X8:
+
+* [Update for OS release V.399](#update-for-os-release-v399)
+* [Update through Out-of-the-box experience](#update-through-out-of-the-box-experience)
+* [Update through Portenta X8 Manager in your Arduino Cloud for Business account (available for all OS releases)](#update-with-portenta-x8-board-manager)
+* [Update using the `uuu` tool (compatible with custom images)](#update-using-uuu-tool)
+
+## Check Portenta X8 OS Release
+
+To verify which OS release is flashed on your Portenta X8, you need to connect to your board through **ADB**.
+
+***If you have not yet read how to work with Linux on the Portenta X8, please check out [this section](https://docs.arduino.cc/tutorials/portenta-x8/user-manual/#working-with-linux) of the Portenta X8 user manual.***
+
+You can type the following command in the command line window to get the OS release currently running on your device.
+
+```bash
+cat /etc/os-release
+```
+
+![Get OS release](assets/adb-shell-os-release.png "Get OS Release")
+
+As shown in the image above, the OS release of this Portenta X8 corresponds to **`IMAGE_VERSION=569`**.
+
+## Update For OS Release V.399
+
+Open a new Command Line window and connect to your Portenta X8. At this point, verify that your Portenta X8 is connected to a network and use the following commands:
+
+```bash
+wget https://downloads.arduino.cc/portentax8image/399-install-update
+```
+
+```bash
+chmod +x 399-install-update
+```
+
+```bash
+sudo ./399-install-update
+```
+
+Now, you need to reboot the board by pressing its pushbutton for around *10 seconds*. After that, connect again to your Portenta X8 through the Command Line and type the following commands:
+
+```bash
+wget https://downloads.arduino.cc/portentax8image/399-finalize-update
+```
+
+```bash
+chmod +x 399-finalize-update
+```
+
+```bash
+sudo ./399-finalize-update
+```
+
+These commands will make your V.399 Portenta X8 compatible with the [aklite-offline](https://docs.foundries.io/latest/user-guide/offline-update/offline-update.html) tool. It will then allow you to update your Portenta X8 to the latest image version of Arduino released.
+
+Arduino provides this tool for free for any Portenta X8 user to enable offline secure updates to all devices, even if those devices are not connected to any *FoundriesFactory*.
+
+When the update process has finished, your Portenta X8 will start running the latest OS release immediately.
+
+***If your Portenta X8 is running _OS release V.399_ and is connected to a compatible carrier board, like the Portenta Breakout, it is recommended to update it to the latest image release by following the instructions in the [flashing with `uuu tool`](#update-using-uuu-tool) section.***
+
+## Update Through Out-Of-The-Box Experience
```

Comment:
Rename this Arduino Linux Wizard everywhere please

Reply:
Updated!